src/@chakra-ui/gatsby-plugin/foundations/colors.ts

```diff
@@ -10,30 +10,35 @@ const colors = {
     900: "#141414",
   },
   blue: {
-    100: "#e8e8ff",
-    300: "#8d8dff",
+    100: "#dedeff",
+    300: "#ababfe",
     500: "#1c1cff",
+    600: "#090990",
     700: "#0b0b66",
   },
   orange: {
     100: "#ffe3d3",
     300: "#ffb991",
     500: "#ff7324",
-    800: "#451900",
+    600: "#c95d20",
+    800: "#6a3301",
   },
   red: {
-    100: "#F7C8C8",
+    100: "#f7c8c8",
     500: "#b80000",
     900: "#1B0C0C",
```

Comment:
I think we are deprecating this color as well, right? I don't see that on the DS anymore. cc @nloureiro

Reply:
This color is being recognized as `Error Light` in the DS

Reply:
I'm talking about the `red.900`. It is not on the DS anymore.
![image](https://user-images.githubusercontent.com/468158/224332544-b1520210-0c9e-4175-9faa-9869426b0a07.png)

Reply:
It is only being used in the `errorNeutral` semantic which I have already marked for deprecation, so this should also be marked.